2016/03/10 18:51:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "dist/img/user4-128x128.jpg".' in C:\wamp\www\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user4-...')
#1 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/plateforme/dist/img/user4-128x128.jpg
HTTP_REFERER=http://localhost/plateforme/
---
2016/03/10 18:51:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "dist/img/user3-128x128.jpg".' in C:\wamp\www\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user3-...')
#1 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/plateforme/dist/img/user3-128x128.jpg
HTTP_REFERER=http://localhost/plateforme/
---
2016/03/10 18:51:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "dist/img/user2-160x160.jpg".' in C:\wamp\www\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user2-...')
#1 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/plateforme/dist/img/user2-160x160.jpg
HTTP_REFERER=http://localhost/plateforme/
---
2016/03/11 11:50:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'plateforme'' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('User')
#8 C:\wamp\www\plateforme\protected\controllers\UserController.php(126): CActiveDataProvider->__construct('User')
#9 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#10 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\wamp\www\Yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\wamp\www\Yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\wamp\www\Yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\wamp\www\Yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\wamp\www\Yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\wamp\www\Yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('')
#19 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#20 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/plateforme/user
HTTP_REFERER=http://localhost:82/plateforme/index.php/site/login
---
2016/03/11 12:00:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\wamp\www\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/plateforme/login
---
2016/03/11 12:02:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\wamp\www\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\wamp\www\plateforme\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/plateforme/index.php/login
---
